keps/sig-autoscaling/2702-graduate-hpa-api-to-GA/README.md

```diff
@@ -0,0 +1,579 @@
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ x ] Events
+  - Event Reason: 
+  The event type `Normal`, reason `SuccessfulRescale`, note `New size: N; reason: FOO` indicates autoscaling is operating normally.
+  Abnormal events type `Warning` include reasons such as `FailedRescale` and `FailedComputeMetricsReplicas` and will
+  include details about the error in the note.
+- [ x ] API .status
+  - Condition name: 
+  There are three condition types which indicate the operating status of the HPA.  They are `ScalingEnabled`, `AbleToScale`
+  and `ScalingLimited` (see type [comments](https://pkg.go.dev/k8s.io/api/autoscaling/v2beta2#HorizontalPodAutoscalerConditionType))
+  Under normal operating circumstances `ScalingEnabled` and `AbleToScale` should be status `true`, indicating the HPA is
+  successfully reconciling the scale. `ScalingLimited` indicates user configuration is limiting the "ideal" scale with a
+  minimum, maximum, rate or delay. Which limit is the cause will be indicated in the message.
+  It's normal for this to be `true` or `false` periodically.
+  - Other field: 
+- [ x ] Other (treat as last resort)
+  - Details:
+  The HPA status includes the current observed metric values, one for each given target. Using these
+  values an operator can verify the HPA is maintaining the desired target for the dominant metric.
+  The operator can also see the number of pods the HPA observed under `status.currentReplicas` and the most
+  recent recommendation under `status.desiredReplicas`.
+  The latest observed generation is echoed back in status so an operator can verify the HPA is keeping up-to-date with
+  configuration changes.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
```

Comment:
I think we should remove this as we currently have no SLIs on the operation of the HPA, and don't intend to add them as part of this graduation currently.

Reply:
Ack, updated it. Thank you.